First pull request for second part of mobile setup portfolio. Html and Css code added.

TechnoAndy opened the pull request:
Pull request for requested changes from first review have been made. Align project image work cards and content.

TechnoAndy committed:
Add html and css

Add html code: works, mps, class, buttons, images
Add css code: properties

TechnoAndy committed:
Add html and css file

Add html and css code.
Complete code added. Had issues with first code had to re-do entire file.

zainsadaqat reviewed (changes requested):
# INVALID ❌ ❌ ❌
# Hello @TechnoAndy  👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where all linters are not passing. Please fix the linter errors and add a description to this pull request as well. 👍🏻 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

<img src="https://media4.giphy.com/media/DN1I1TGhL83cNx9DSR/giphy.gif"/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

TechnoAndy committed:
Update index.html

Fix linter errors

TechnoAndy committed:
Update style.css

Fix linters errors

TechnoAndy committed:
Update style.css

Update linter

TechnoAndy committed:
Update index.html

Update linter errors

TechnoAndy committed:
html linter errors

Update linter error

TechnoAndy committed:
Fix linter errors

Fix linter errors

TechnoAndy committed:
Update linter error

Removed incorrect file name added whitespace

TechnoAndy committed:
Update index.html

Fix linter errors

TechnoAndy committed:
Linter errors

Fixed linter errors

MeqdamAlqudah commented on index.html:
```diff
@@ -12,7 +12,7 @@
     <header class="navbar">
         <a href="#" class="nav-logo">My Logo</a>
```
- [ ] Please fix this margin so your design looks like the design , please align the items in the works section using grid
![image](https://user-images.githubusercontent.com/74744787/185764888-226135c8-daa9-4c8c-ac58-121b9a01ffdc.png)

MeqdamAlqudah commented on index.html:
```diff
@@ -29,10 +29,186 @@ <h1 class="content-text">I'm Andrea Glad to see you!</h1>
+            </section>
+            <section class="works">
+                <div class="container">
+                  <div class="work-card">
+                  <div>
+                      <img src="SnapshootPortfolio01.png" alt="" />
+                    </div>
```
- [ ] Please align the items in each card using flexbox
![image](https://user-images.githubusercontent.com/74744787/185764971-2952af03-0ab7-4836-8eb1-7fb0f88284ac.png)

MeqdamAlqudah commented on index.html:
```diff
@@ -29,10 +29,186 @@ <h1 class="content-text">I'm Andrea Glad to see you!</h1>
+                  </div>
+              </section>
+              <section class="about-section-2">
+                <div class="align-center-text">
+                  <h1 class="header-text">About<br>Myself</h1>
```
- [ ] Please use only one h1 tag in the HTML doc this can be powerful for the SEO.

TechnoAndy replied:
Change has been made as requested. Thank you.

MeqdamAlqudah commented on index.html:
```diff
@@ -29,10 +29,186 @@ <h1 class="content-text">I'm Andrea Glad to see you!</h1>
+                        <li class="works-badge">javascript</li>
+                      </ul>
+                      <button class="btn" type="button">See Project</button>
+                    </div>
+                  </div>
+                  </div>
+              </section>
+              <section class="about-section-2">
+                <div class="align-center-text">
+                  <h1 class="header-text">About<br>Myself</h1>
```
- [ ] Please align the items in the about me section using flexbox
![image](https://user-images.githubusercontent.com/74744787/185765135-d07b42ba-076e-4120-be1d-bee6c5db6bdc.png)

MeqdamAlqudah reviewed (changes requested):
Hi @TechnoAndy , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![](https://media4.giphy.com/media/7SyGkO0kQWCfnUC47Y/giphy.gif?cid=ecf05e47ou0itf7x5gr3sb3quhztib88pv00acxurgbvq0dl&rid=giphy.gif&ct=g)

 ## Highlights
:+1:  Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TechnoAndy committed:
Review changes

Made changes requested by reviewer. Flexbox and grid alignment.

TechnoAndy committed:
Update style.css

Linter errors

TechnoAndy committed:
Update style.css

Update linter errors

TechnoAndy committed:
Update style.css

Update linter errors

TechnoAndy committed:
Update style.css

Update linter errors

TechnoAndy committed:
Update style.css

Fix linter error

TechnoAndy committed:
Update style.css

Fix linter

TechnoAndy committed:
Update style.css

Fix linter errors

TechnoAndy committed:
Update index.html

Fix linter error in html

TechnoAndy committed:
Update index.html

Attempt fix html linter error

TechnoAndy committed:
Update index.html

Update linter error

TechnoAndy committed:
Update index.html

Updated linter error

TechnoAndy committed:
Update index.html

Update error

TechnoAndy committed:
HR

Updated changes from linter errors

TechnoAndy committed:
Html changes

Update html code

TechnoAndy committed:
Update style.css

Linter error, unexpected whitespaces

TechnoAndy committed:
Update index.html

Updated linter changes

TechnoAndy committed:
Update index.html

Update linter error

TechnoAndy committed:
Update index.html

Update linter errors

TechnoAndy committed:
Update index.html

Update linter errors

TechnoAndy committed:
Update code

Change made to Html and Css code to get it looking more like the Figma template.

TechnoAndy committed:
Linter errors

Update linter errors

TechnoAndy committed:
Linter errors

Update to linter errors

TechnoAndy committed:
Linter error

update linter errors

TechnoAndy committed:
Linter errors

Update linter errors

TechnoAndy committed:
Linter errors

Update linter errors

levy002 commented on style.css:
```diff
@@ -120,12 +125,221 @@
+  width: 300px;
+  margin: 0;
+  margin-left: 13px;
+}
```
- [ ] Kindly make sure to align your content in this `about myself` section with CSS `flex-box` which will help you to maintain your website responsiveness easily and will make your work looks more professional :wink:

levy002 commented on style.css:
```diff
@@ -120,12 +125,221 @@
   background-color: transparent;
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  gap: 50px;
+}
```
- [ ] Kindly make sure your project cards are placed in the center of the page. Go ahead and just your margins and padding in order to stick to the provided Figma design. :wink:

  - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/185867571-883959e4-8047-405d-878d-083948ea96c2.png)


  - ### Figma design
  ![image](https://user-images.githubusercontent.com/87197412/185867239-8da68725-41f5-45ad-a4e9-a1237cf8c85a.png)

levy002 commented on style.css:
```diff
@@ -120,12 +125,221 @@
+  margin-left: 13px;
+}
+
+.work-card-img {
+  object-fit: cover;
+  align-items: center;
+  justify-content: center;
+  margin-top: 12px;
+}
```
- [ ] Kindly make sure your project image place holder is aligned exactly as how it looks in the provided Figma design. Please adjust the image width. :wink: 
 
   - ### Your design
   
  ![image](https://user-images.githubusercontent.com/87197412/185870643-675e5a73-5cc9-48d2-a9c5-a9d3b9272990.png)


   - ### Figma design
  
  ![image](https://user-images.githubusercontent.com/87197412/185870705-565c46f3-55d1-4e3f-8320-643d515861b3.png)

levy002 commented on style.css:
```diff
@@ -120,12 +125,221 @@
+  margin: 0;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 7px;
+  border-color: var(--color-primary);
+  background-color: transparent;
+}
+
```
- [ ] Kindly implement all button interactions (enable, disabled, active) as it is requested in project requirements. It will increase user experience as well :wink:

   - ### Figma design
   
   ![image](https://user-images.githubusercontent.com/87197412/185871944-d0690fa4-c18e-4682-9da2-9121d8da96d1.png)

levy002 reviewed (changes requested):
## Hello @TechnoAndy   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._
 - [ ] Kindly remove this `.vscode` file from your repo in order to make our code base clean. :smiley: 

   ![image](https://user-images.githubusercontent.com/87197412/185872742-4a69bbe1-6c4e-48d2-8341-7b8093973051.png)

 - [ ] You did a good job on trying to follow the Microverse README template. However, It would be better if you clone this Microverse README template and customize it to personalize it according to the project. Please go ahead and copy this [README template ](https://github.com/microverseinc/readme-template)and customize it.
   
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TechnoAndy committed:
Delete settings.json

Removed .vscode file as requested by code reviewer

TechnoAndy committed:
Update README.md

Updated correct template for README file

TechnoAndy committed:
Merge branch 'main' into add-second-mobile-setup

TechnoAndy committed:
Merge branch 'main' into add-second-mobile-setup

TechnoAndy committed:
Media tabs

Changed order of media tabs

TechnoAndy committed:
Merge branch 'add-second-mobile-setup' of https://github.com/TechnoAndy/My-Portfolio into add-second-mobile-setup

TechnoAndy committed:
Update index.html

Linter error updates

TechnoAndy committed:
Update style.css

Update linter error

TechnoAndy committed:
Update style.css

Linter error update

TechnoAndy committed:
Update style.css

Change content alignment in about section

TechnoAndy committed:
Linter errors css

Update linter errors

TechnoAndy committed:
Update style.css

Remove duplicate margin

TechnoAndy committed:
Update style.css

Update linter errror

TechnoAndy committed:
Update style.css

Update linter errors

TechnoAndy committed:
Update style.css

Update linter error

TechnoAndy committed:
2nd review

Made changes as requested from the reviewer.

TechnoAndy committed:
Update style.css

Update changes for word card img

TechnoAndy committed:
Update style.css

Update linter error

TechnoAndy committed:
Update style.css

Updated linter changes

TechnoAndy committed:
Update style.css

Updated review changes

TechnoAndy committed:
Update linters.yml

Removed file

TechnoAndy committed:
Update linters.yml

Add code file

Bondok6 reviewed (commented):
## Hi @TechnoAndy 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Your Pull Request should have a short summary describing introduced changes in more detail. Please add a summary 👍

![image](https://user-images.githubusercontent.com/45231276/186111414-e998265e-127e-40eb-ba36-29725824cc6e.png)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

TechnoAndy commented:
Pull request for requested changes from first review have been made. Align project image work cards and content.

sumairq reviewed (approved):
### Hi @TechnoAndy  , 

<img src="https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TechnoAndy committed:
Update style.css

Changed font weight

TechnoAndy committed:
Update style.css

Change font weight
